app/columns.tsx

```diff
@@ -229,11 +230,11 @@ export const experimentColumns: ColumnDef<RecipeOrBranchInfo>[] = [
         // XXX should figure out how to do this NimbusRecipe instantiation
         // once per row (maybe useState?)
         const recipe = new NimbusRecipe(props.row.original.nimbusExperiment)
-        const branchLink = recipe.getBranchScreenshotsLink(
-            props.row.original.slug);
-        return (
-          OffsiteLink(branchLink, "Screenshots")
-        )
+        const branchLink =
+          props.row.original.screenshots.length > 0
+            ? recipe.getBranchScreenshotsLink(props.row.original.slug)
+            : null;
```

Comment:
Please break this up into a few lines (maybe just use an `if` rather than the ternary operator, so that it's easier to read because one doesn't have to make so many mental context switches in a single statement.